game/scripts/npc/items/item_crimson_guard.txt

```diff
@@ -96,7 +96,7 @@
       "08"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_aoe_armor"                                 "2 3 4 5 6"
+        "bonus_aoe_armor"                                 "2 4 6 8 10"
```

Comment:
I would leave the AOE bonus armour low as your also getting alot of physical damage block for your team.

Reply:
"Alot" LUL

Reply:
Its dispellable, I dont see the issue.